crates/tauri-utils/src/config.rs

```diff
@@ -1773,6 +1773,404 @@ pub struct AssetProtocolConfig {
   pub enable: bool,
 }
 
+/// A capability entry which can be either an inlined capability or a reference to a capability defined on its own file.
+#[derive(Debug, Clone, PartialEq, Serialize)]
+#[cfg_attr(feature = "schema", derive(JsonSchema))]
+#[serde(untagged)]
+pub enum CapabilityEntry {
+  /// An inlined capability.
+  Inlined(Capability),
+  /// Reference to a capability identifier.
+  Reference(String),
+}
+
+impl<'de> Deserialize<'de> for CapabilityEntry {
+  fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
+  where
+    D: Deserializer<'de>,
+  {
+    UntaggedEnumVisitor::new()
+      .string(|string| Ok(Self::Reference(string.to_owned())))
+      .map(|map| map.deserialize::<Capability>().map(Self::Inlined))
+      .deserialize(deserializer)
+  }
+}
+
+/// The application pattern.
+#[skip_serializing_none]
+#[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "lowercase", tag = "use", content = "options")]
+#[cfg_attr(feature = "schema", derive(JsonSchema))]
+pub enum PatternKind {
+  /// Brownfield pattern.
+  Brownfield,
+  /// Isolation pattern. Recommended for security purposes.
+  Isolation {
+    /// The dir containing the index.html file that contains the secure isolation application.
+    dir: PathBuf,
+  },
+}
+
+impl Default for PatternKind {
+  fn default() -> Self {
+    Self::Brownfield
+  }
+}
```

Comment:
Let's move these back to where they were so the git diff of this PR doesn't include any changes to them.